index.js

```diff
@@ -60,11 +61,11 @@ function setEventNames(sqsConfig) {
   );
 }
 
-async function sendToSqs({ record, params }) {
+async function sendToSqs({ record, params, logPayloadTransformer = _.identity }) {
   const message = params.bodyHandler(record);
   const MessageBody = JSON.stringify(message);
 
-  params.logger.info('DynamoDB Record: %j', record);
+  params.logger.info('DynamoDB Record: %j', logPayloadTransformer(record));
```

Comment:
I think we should try-catch this. It's really, really important that this stream handler doesn't throw and a logic error in the log transformer could easily happen.

Reply:
Right. We'll now log the record without the transformer in the catch so we can debug those errors